bindings/py/tests/sdr/SDR_test.py

```diff
@@ -444,27 +424,25 @@ def testSparsity(self):
             ( 0.5,  0.5,  0.5, 0.5),
             ( 0.11, 0.25, 0.33, 0.5, 0.98, 0.98, 0.98, 0.98, 0.98, 0.98, 0.98),
         ]
+        size = 10000
         seed = 99
+        X    = SDR( size )
         for sparsities in test_cases:
             sdrs = []
             for S in sparsities:
-                inp = SDR( 10000 )
+                inp = SDR( size )
                 inp.randomize( S, seed)
                 seed += 1
```

Comment:
btw, OT< nit: it's ok to keep seed the same

Reply:
The seed needs to be different so that every random SDR is different.  Otherwise two random SDRs with the same size & sparsity will have the exact same bits, and so they will have 100% overlap, and this unit test will fail.

Reply:
> [The seed needs to be different] so that every random SDR is different.  

really? That is how randomize should work. I haven't tested this, but the RNG lives out of scope of randomize, so it keeps state between random() calls... oh, ok! It's a seed, not rng! 
Maybe we should change that to a rng instance? but another PR

Reply:
#381